bellman/src/lib.rs

```diff
@@ -23,11 +21,9 @@ extern crate hex_literal;
 #[cfg(test)]
 extern crate rand;
 
-#[cfg(test)]
-extern crate rand_xorshift;
```

Comment:
Why does this `test`-conditional `extern crate` disappear, but not the `extern crate rand`?